src/Service.Tests/SqlTests/RestApiTests/Put/MsSqlPutApiTests.cs

```diff
@@ -233,6 +251,81 @@ public async Task PutOneInViewBadRequest()
             await base.PutOneInViewBadRequest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate that even when an update DML trigger is enabled on a table, we still return the
+        /// latest data as it is present after the trigger gets executed. Whenever an update DML trigger is enabled,
+        /// we use a subsequent SELECT query to get the data instead of using OUTPUT clause.
+        /// </summary>
+        [TestMethod]
+        public async Task PutOneUpdateInTableWithUpdateTrigger()
+        {
+            // Validate that PUT operation (resulting in update) succeeds when an update DML trigger is enabled for a table
```

Comment:
This can be two different tests, would be easier to debug incase of failures.

Reply:
The intent of the test is to validate behavior of PUT update on tables. Autogen/Non-autogen are just two different cases. I think we should keep this as is.